modoboa/admin/migrations/0001_initial.py

```diff
@@ -19,7 +19,6 @@ class Migration(migrations.Migration):
             ],
             options={
                 'ordering': ['address'],
-                'permissions': (('view_aliases', 'View aliases'),),
```

Comment:
Old migrations must remain untouched, otherwise fresh installations won't behave like new ones and it could lead to unexpected migration issues.

Reply:
I don't think there is an impact with this migration change. I have removed them for at least two reasons:
* as `view_aliases` - and others - is finally renamed to `view_alias` in the last migration, and because permissions are created at the end of the migration process, it prevents to create the `view_aliases` permission without renaming it on fresh installations.
* for `view_domain`, it prevents a conflict starting with Django 2.1 where this permission will be automatically created by Django. If you try to keep this one in the migration and run `./manage.py migrate`, you will end with a `UNIQUE constraint failed: auth_permission.content_type_id, auth_permission.codename`.

Reply:
Ok.